api/src/main/java/org/openmrs/module/fhir/api/impl/base/BaseConditionHelperImpl.java

```diff
@@ -59,6 +61,13 @@ public Condition createCondition(Condition condition) {
 		return FHIRConditionUtil.generateFHIRConditionFromOpenMRSObs(obs);
 	}
 
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#getConditionsByPatientUuid(java.lang.String)
+	 */
+	public List<Condition> getConditionsByPatientUuid(String patientUuid) {
+		throw new NotImplementedException();
```

Comment:
Did you intentionally decide not to implement this?

Reply:
Yeah,
condition service was introduced since openmrs version 2.2. So for the previous version conditions were save as obs.  Which complicates implementation for that. When using ObsService to all obs by person. need to know which are conditions, that are active,
 which inactive. Basically still need the decision to be made on this.
 For openmrs 2.2.* and above, It's working fine.

Reply:
Fair enough! 😊